scss/layout/_section.scss

```diff
@@ -1,6 +1,8 @@
 @import "./../settings";
 @import "./../functions";
 @import "./../mixins";
+@import "./../core/root";
```

Comment:
Why is this being imported?

Reply:
there are some css variables like `--fd-padding-ui` being defined there